### DISCUSSION TITLE
Does not work as standalone / without internet?

### DISCUSSION THREAD

**Discussion by feltz916** [vicwomg/pikaraoke#366] [General]
Before i write up a long bug, i'm curious if this new structure was intended to break standalone functionality?

I previously was able to run the software with no internet connection, and even setup my device as an access point so i can queue songs remotely.

Now, i get one of two behaviors:
With no internet setup, it just hangs at a white screen. (invalid argument exception in webdriver)
With no internet, and AP enabled, it just crashes (typeerror in dispatch request)

**Comment by vicwomg** [comment]
No, that is not the expected behavior. I just tested offline use myself on
a macbook and it worked fine, but it's possible there's something wrong
with the pi. Feel free to file a bug, but that invalid argument exception
doesn't sound like an issue with networking and seems more related to
chromedriver / chromium. Did you confirm it works with an internet
connection?

**Reply by feltz916** [reply]
yes it works fine with internet. I'll write up a bug now.
